README.md

```diff
@@ -585,6 +586,30 @@ Arguments:
 ...
 ```
 
+#### get_columns ([source](macros/sql/get_columns.sql))
+This macro returns the unique columns for a given [relation](https://docs.getdbt.com/docs/writing-code-in-dbt/class-reference/#relation) as a comma-separated list.
```

Comment:
```suggestion
This macro returns the unique columns for a given [relation](https://docs.getdbt.com/docs/writing-code-in-dbt/class-reference/#relation) as a Jinja list
```

Right? From what I can see it's returning an iterable list of items, not a comma-separated values that you can drop directly into a query.

Reply:
Yes, correct. I was thinking in python terms, it returns a list object.
I reworded that to make it more clear.